HACKING.adoc

```diff
@@ -254,3 +249,33 @@ Then proceed as documented in the `README.adoc`.
 . Update version in `lib/asciidoctor-revealjs/version.rb` and `package.json` (+1 bugfix and append '-dev') and commit
 ** commit msg: Begin development on next release
 
+== Ruby and asciidoctor-doctest tests
+
+=== Running tests
+
+We recommend tests to be run with a fresh install of all dependencies in a local folder that won't affect your ruby install (a `.bundle/` in this directory):
+
+    bundle --path=.bundle/gems --binstubs=.bundle/.bin
+
+Then you can execute the tests with:
+
+    bundle exec rake test
+
+However, if you have all dependencies properly installed this command should run the tests successfully:
+
+    rake test
+
+=== Generating HTML test target
+
+Tests were bootstrapped by generating them from asciidoctor-doctest's test corpus and current asciidoctor-revealjs' slim template engine.
+This is done using the following command:
+
+    bundle exec rake generate
```

Comment:
Maybe add note about `FORCE=y` for updating changed examples and add link to https://github.com/asciidoctor/asciidoctor-doctest/#generate-examples ?